core/ai_worker.go

```diff
@@ -358,6 +358,12 @@ func (rw *RemoteAIWorker) Process(logCtx context.Context, pipeline string, model
 	}
 
 	clog.V(common.DEBUG).Infof(logCtx, "Job sent to AI worker worker=%s taskId=%d pipeline=%s model_id=%s", rw.addr, taskID, pipeline, modelID)
+	monitor.SendQueueEvent("ai_worker_job_sent", map[string]string{
```

Comment:
Let's make `SendQueueEvent()` call async and name it `SendQueueEventAsync()` to be specific that this call never blocks. And I believe it should never block.